packages/core/src/connectors/facebook/constant.ts

```diff
@@ -0,0 +1,4 @@
+export const authorizationEndpoint = 'https://www.facebook.com/v13.0/dialog/oauth';
+export const accessTokenEndpoint = 'https://graph.facebook.com/v13.0/oauth/access_token';
+export const userInfoEndpoint = 'https://graph.facebook.com/v13.0/me';
```

Comment:
per https://developers.facebook.com/docs/graph-api/reference/user/, I don't think this is a fixed URL, please double check

Reply:
Reference: https://developers.facebook.com/docs/graph-api/overview#me

> The /me node is a special endpoint that translates to the object ID of the person or Page whose access token is currently being used to make the API calls. If you had a User access token, you could retrieve a User's name and ID by …

Reply:
per the link you post, the endpoint should be `https://graph.facebook.com/me`?
It is confusing that the documents are not consistent. We need to access this API manually to check its correctness.

Reply:
Yes. Updated to `https://graph.facebook.com/me` for the consistency with the documentation. PTAL.

Reply:
Tested. They're both working.

https://developers.facebook.com/docs/graph-api/overview#versions

> If you do not include a version number we will default to the oldest available version, so it's recommended to include the version number in your requests.

Added the necessary comments in `facebook/constant.ts`.

```
/**
 * Note: If you do not include a version number we will default to the oldest available version, so it's recommended to include the version number in your requests.
 * https://developers.facebook.com/docs/graph-api/overview#versions
 */
export const authorizationEndpoint = 'https://www.facebook.com/v13.0/dialog/oauth';
export const accessTokenEndpoint = 'https://graph.facebook.com/v13.0/oauth/access_token';
/**
 * Note: The /me node is a special endpoint that translates to the object ID of the person or Page whose access token is currently being used to make the API calls.
 * https://developers.facebook.com/docs/graph-api/overview#me
 * https://developers.facebook.com/docs/graph-api/reference/user#Reading
 */
export const userInfoEndpoint = 'https://graph.facebook.com/v13.0/me';
export const scope = 'email,public_profile';
```